Signal/src/util/OWSBackupImportJob.m

```diff
@@ -20,36 +20,14 @@
 
 #pragma mark -
 
-@interface OWSBackupImportItem : NSObject
```

Comment:
This class got promoted to the base "job" class since now both import and export jobs need to download and parse fragments from the last backup.